django_esm/finders.py

```diff
@@ -29,9 +30,25 @@ def _check_package_json(self):
             ]
         return []
 
-    def find(self, path, all=False):
+    def _check_deprecated_find_param(self, find_all=False, **kwargs):
+        if "all" in kwargs:
+            warnings.warn(
+                "The 'all' argument of the find() method is deprecated in favor of "
+                "the 'find_all' argument.",
+                category=DeprecationWarning,
```

Comment:
If you use the right warning, we will remember to delete the extra code.
```suggestion
                category=RemovedInDjango61Warning,
```